### ISSUE TITLE
Is it a bug?

### ISSUE THREAD

**Issue by NAlexandrov** [eviltik/cidr-clean#11]
Current behavior:
```js
cidrClean([
  '192.168.1.1/32',
  '192.168.1.0/30',
  '192.168.0.0/16'
]); // returns [];
```

Expected behavior:
```js
cidrClean([
  '192.168.1.1/32',
  '192.168.1.0/30',
  '192.168.0.0/16'
]); // returns ['192.168.0.0/16'];
```

**Comment by eviltik** [comment]
yep it's a bug, let me take a look

**Comment by eviltik** [comment]
fixed, thx for feedback, bump version 1.1.7
